src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -20,12 +20,16 @@
 
 /**
  * @author synopia
+ * Representation of a region or floor
  */
 public class BitMap {
     public static final int KERNEL_SIZE = 3;
     public static final float SQRT_2 = (float) Math.sqrt(2);
     BitSet map;
 
+    /**
+     * Creates a new Bitmap
```

Comment:
- what properties does this bitmap have if created this way?
- is it empty? 
- does it have a size?
- what is the default bit value?